src/main/java/org/spongepowered/common/mixin/core/command/MixinSubject.java

```diff
@@ -63,7 +63,7 @@
  */
 @NonnullByDefault
 @Mixin(value = {EntityPlayerMP.class, TileEntityCommandBlock.class, EntityMinecartCommandBlock.class, MinecraftServer.class, RConConsoleSource.class,
```

Comment:
Why is `TileEntityCommandBlock` (and EntityMinecartCommandBlock, I guess) a Subject ?

Reply:
I'd like to know too, but I don't know enough about permissions to know what should be a Subject. 
I don't want to make this PR go beyond what it was meant for when I opened it - I don't want to make (breaking?) changes in this one, especially since I don't have enough knowledge about the Permissions API.

Reply:
As discussed the other day, `TileEntityCommandBlock` and `EntityMinecartCommandBlock` should not be here. This should mixin into `CommandBlockBaseLogic.class`. Just like TileEntitySign is not the IMixinSubject.